open_instruct/grpo_vllm_thread_ray_gtrl.py

```diff
@@ -985,7 +1017,7 @@ def generate_with_engines(prompts: List[List[int]], sampling_params: SamplingPar
         queries_next = data[INPUT_IDS_PROMPT_KEY].to(device)
         ground_truths_next = data[GROUND_TRUTHS_KEY]
         datasets_next = data[DATASET_SOURCE_KEY]
-        if accelerator.is_main_process:
+        if self.rank == 0:
```

Comment:
is there a reason for this over accelerator main process?

Reply:
The `accelerator.is_main_process` is kind of a hack. Since we now extensively using deepspeed stack, I wanted to clean it up a bit.